tests/test_genetic_maps.py

```diff
@@ -171,6 +184,59 @@ def test_multiple_threads_downloading(self):
             gm.is_cached = saved
 
 
+class TestGeneticMapDownload_security(tests.CacheWritingTest):
```

Comment:
Class names are CamelCase according to pep8. Might become something that generates pep8 failures in the future, so might as well stick to the conventions.